bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/SimpleRow.java

```diff
@@ -0,0 +1,124 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.grpc;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Ordering;
+import com.google.protobuf.ByteString;
+
+/**
+ * <p>This class stores values from {@link com.google.bigtable.v2.ReadRowsResponse} objects and
+ * represents a single row.</p>
+ *
+ * @author tyagihas
+ * @version $Id: $Id
+ */
+public class SimpleRow {
+  public final class SimpleColumn {
+    private final String family;
+    private final ByteString qualifier;
+    private final long timestamp;
+    private final ByteString value;
+    private final List<String> labels;
+
+    private SimpleColumn(String family, ByteString qualifier, long timestamp, ByteString value,
+        List<String> labels) {
+      this.family = family;
+      this.qualifier = qualifier;
+      this.timestamp = timestamp;
+      this.value = value;
+      this.labels = labels;
+    }
+
+    public String getFamily() {
+      return family;
+    }
+
+    public ByteString getQualifier() {
+      return qualifier;
+    }
+
+    public long getTimestamp() {
+      return timestamp;
+    }
+
+    public ByteString getValue() {
+      return value;
+    }
+
+    public List<String> getLabels() {
+      return labels;
+    }
+  }
+
+  public static class FamilyColumnOrdering extends Ordering<SimpleColumn> {
+    public static final FamilyColumnOrdering DEFAULT_ORDERING = new FamilyColumnOrdering();
+
+    private FamilyColumnOrdering() {}
+
+    @Override
+    public int compare(SimpleColumn left, SimpleColumn right) {
+      int result = left.family.compareTo(right.family);
+      if (result == 0) {
+        result = left.qualifier.toStringUtf8().compareTo(right.qualifier.toStringUtf8());
+      }
+      return result;
+    }
+  }
+
+  public static final class Builder {
+	private static Builder DEFAULT_INSTANCE = new Builder();
+	private final ImmutableList.Builder<SimpleColumn> listBuilder;
+
+    public static Builder getDefaultInstance() {
+      return DEFAULT_INSTANCE;
+    }
+
+    private Builder() {
+      listBuilder = new ImmutableList.Builder<SimpleColumn>();
+    }
+
+    public SimpleRow createRow(ByteString rowKey) {
+      return new SimpleRow(rowKey);
+    }
+  }
+
+  private final ByteString rowKey;
+  private ImmutableList<SimpleColumn> columns = null;
+
+  private SimpleRow(ByteString rowKey) {
+	  this.rowKey = rowKey;
+  }
+  
+  public ByteString getRowKey() {
+  	return rowKey;
+  }
+	
+  public void addCell(String family, ByteString qualifier, long timestamp, ByteString value,
+      List<String> labels) {
+    Builder.getDefaultInstance().listBuilder.add(
+      new SimpleColumn(family, qualifier, timestamp, value, labels));
+  }
+  
+  public ImmutableList<SimpleColumn> getList() {
```

Comment:
This should just be a simple method that returns the simpleColumn immutable List.  Also, there is a preference by Dataflow to use standard java objects like List and Set rather than guava classes.